.github/workflows/pr_ci.yml

```diff
@@ -155,15 +159,22 @@ jobs:
     name: validate all changed modules
     needs: tf_prereqs
     if: ${{ needs.tf_prereqs.outputs.validate_paths != '' }}
-    uses: ./.github/workflows/_tf_validate.yml
+    uses: ./.github/workflows/_tf_test.yml
+    permissions:
+      contents: read
+      id-token: write
     with:
+      cloud: azure
       tf_version: ${{ inputs.tf_version }}
       paths: ${{ needs.tf_prereqs.outputs.validate_paths }}
-      max_parallel: ${{ inputs.max_parallel }}
+      terratest_action: Validate
```

Comment:
What is the reason to start `validate` from uppercase `Validate` ? In Terraform command we have `validate` - can we follow that convention ?

Reply:
👍